kitchenSink/src/jsMain/kotlin/flexBoxDemo.kt

```diff
@@ -68,7 +70,7 @@ fun RenderContext.flexBoxDemo(themeStore: ThemeStore, themes: List<ExtendedTheme
                     left { "0" }
                 }
             }) {
-                themeStore.data.map { currentThemeIndex ->
+                ThemeStore.data.map { currentThemeIndex ->
```

Comment:
Is it really better style to rely on a global variable here?
Passing the store instead of the Handler is imho clearer, as now one dependency is only hidden and the user has to cope with an extra Flow that otherwise was easier to identify!

Reply:
As time is short, I consider thsi a minor issue to discuss.
So no changes for now ;-)